blas/gonum/gonum.go

```diff
@@ -74,3 +74,8 @@ func blocks(dim, bsize int) int {
 func dcabs1(z complex128) float64 {
 	return math.Abs(real(z)) + math.Abs(imag(z))
 }
+
+// conj returns the complex conjugate of x.
+func conj(z complex128) complex128 {
```

Comment:
I know, I thought that having it here might help with inlining. Anyway, removed and call cmplx version instead.